docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1076,7 +1076,10 @@ Query Parameters:
 -   **dockerfile** - path within the build context to the Dockerfile
 -   **t** – repository name (and optionally a tag) to be applied to
         the resulting image in case of success
--   **remote** – git or HTTP/HTTPS URI build source
+-   **remote** – git or HTTP/HTTPS URI build source. When this references
+        a Dockerfile then the `dockerfile` parameter will be ignored. 
+		Additionally, its contents will be placed into a file called 
+		`Dockerfile`.
```

Comment:
**dockerfile** - path within the build context to the Dockerfile. This is ignored if **remote** is specified and points to an individual filename.
**remote** -  A Git repository URI or HTTP/HTTPS URI build source.  If the URI specifies a filename, the file's contents are placed into a file called `Dockerfile`.